modules/markup/reStructuredText/reStructuredText.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package rst
+
+import (
+	"bufio"
+	"bytes"
+
+	"code.gitea.io/gitea/modules/markup"
+
+	gorst "github.com/hhatto/gorst"
+)
+
+func init() {
+	markup.RegisterParser(Parser{})
+}
+
+// Parser implements markup.Parser for reStructuredText
+type Parser struct {
+}
+
+// Name implements markup.Parser
+func (Parser) Name() string {
+	return "reStructuredText"
+}
+
+// Extensions implements markup.Parser
+func (Parser) Extensions() []string {
+	return []string{".rst"}
+}
+
+// Render renders reStructuredText rawbytes to HTML
```

Comment:
Is not `rawbytes` in comment redudant? And should be unified with `RenderString` comment.